### ISSUE TITLE
Create alt text for logic illustrations on extension guidelines

### ISSUE THREAD

**Issue by misolori** [microsoft/vscode-docs#5148]
We currently have no alt text support for these logic screens that we need to figured out the best way to display the info cc @isidorn 

https://code.visualstudio.com/api/references/extension-guidelines#notifications

![image](https://user-images.githubusercontent.com/35271042/152845997-25479d0e-20ef-4f37-99af-38e41ea446fd.png)

**Comment by isidorn** [comment]
Good catch. I suggest that the alt text simply describes this diagram in words
"Show a modal dialog if"
"Show a multi step quick pick if"

If you want I can write this down. Let me know!
Assigning to Feb since I think it is low-haning.

**Comment by misolori** [comment]
@isidorn yea if you want to start that, it would be a huge help! Thanks!
